File: ARCADE/The Core/Level 1_4: seatsInTheater.cs
    using System;
     
    public class Test
    {
    	public static int seatsInTheater(int nCols, int nRows, int col, int row){
            return (nCols - col + 1) * (nRows - row);
        }
    	public static void Main()
    	{
    		 int nCols = 16, nRows = 11, col = 5, row = 3;
    		 Console.Write(seatsInTheater(nCols, nRows, col, row));
    	}
    }

File: ARCADE/The Core/Level 1_4: seatsInTheater.pas
program seatsInTheater;

function seatsInTheater(nCols: integer; nRows: integer; col: integer; row: integer): integer;
begin
    seatsInTheater := (nCols - col + 1) * (nRows - row);
end;

var nCols, nRows, col, row: integer;
begin
	nCols := 16; nRows := 11; col := 5; row := 3;
	writeln(seatsInTheater(nCols, nRows, col, row));
end.
